cns/common/service.go

```diff
@@ -35,7 +35,7 @@ type ServiceAPI interface {
 type ServiceConfig struct {
 	Name        string
 	Version     string
-	Listener    *acn.Listener
+	Listeners   *[]acn.Listener
```

Comment:
this should be []*acn.Listener. We want a list of Listener's pointers, not a pointer to a list of listeners.